code/game/objects/items/weapons/grenades/sonic.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/grenade/sonic
 	name = "SG \"Loudmouth\""
-	desc = "A hailer overclocked to reproduce a noise similar to the leading roaches."
+	desc = "A power cell wired into a short circuit to reproduce a noise similar to the leading roaches."
```

Comment:
Perhaps make it use a circuit then? It needs at least some component that is semi-rare

Reply:
I mean it's not like large power cells are super common either

Reply:
so are circuit boards if you know places.